control-plane/aegisops_control_plane/actions/execution_coordinator_delegation.py

```
@@ -0,0 +1,622 @@
+                    transitioned_at=delegated_at,
+                )
+        except Exception as exc:
+            try:
+                self._record_dispatch_finalization_failure(
+                    execution=predispatch_execution,
+                    execution_run_id=execution_run_id,
+                    finalized_provenance=finalized_provenance,
+                    error=exc,
+                )
+            except Exception:
+                pass
```

Ruff (0.15.12)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)